docs/app/pages/components/components-sections/notifications/notifications/notifications.component.html

```diff
@@ -0,0 +1,104 @@
+            <p class="notification-title">Messages Received</p>
+            <p class="notification-label">You have 16 messages</p>
+            <small class="notification-subtitle">{{notificationRef.date | date:'EEEE, MMMM d, y, h:mm:ss a'}}</small>
+        </div>
+    </div>
+</ng-template>
+
+<hr>
+
+<!-- <div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <accordion>
+            <accordion-group heading="Customize Example...">
+                <div class="row uxd-customize-row">
+                        <div class="col-md-4 col-sm-12">
+                            <label for="duration">Duration</label>
+                            <ux-number-picker [(value)]="notificationDuration" min="0" max="3600000"></ux-number-picker>
+                        </div>
+                    </div>
+                </accordion-group>
+        </accordion>
+    </div>
+</div> -->
+
+<p>Notifications can be used to inform the user of occurring events or to alert the user of something that may require their attention.
+    The <code>ux-notification-list</code> component can be used to display a list of notifications.</p>
+
+<p>The following attributes can be used to customize the <code>ux-notification-list</code> component:</p>
+
+<uxd-api-properties tableTitle="Notification List">
+    <tr uxd-api-property name="position" type="string" defaultValue="top-right">
+        Sets the position of the list of notifications within the browser window.
+        The possible options are: <code>top-right</code>, <code>top-left</code>, <code>bottom-left</code> and <code>bottom-right</code>.
+    </tr>
+    <tr uxd-api-property name="direction" type="string" defaultValue="above">
+        Sets the order in which notifications are displayed:
+      <ul>
+        <li><code>above</code> - newer notifications will appear above older ones.</li>
+        <li><code>below</code> - newer notifications will appear below older ones.</li>
+      </ul>
+    </tr>
+</uxd-api-properties>
+
+<p>Notifications are controlled by the <code>notificationService</code>. It is recommended that the service is injected into the <code>AppComponent</code>.
+    The following functions can be called on the service:</p>
+
+<uxd-api-properties tableTitle="Notification Service">
+    <tr uxd-api-property name="show">
+        This function should be called to show a notification.
+        It should be given a <code>TemplateRef</code> containing the content to be displayed and optionally an object containing properties
+        to configure the notification.
+        It will return a <code>NotificationRef</code> object which will contain the information passed to the <code>showNotification</code> function
+        along with a <code>date</code> property which will contain the time at which the notification was shown.
+    </tr>
+    <tr uxd-api-property name="getHistory">
+        This function will return an array of <code>NotificationRef</code> objects.
```

Comment:
Possibly mention that this returns an array containing all the notifications that have been shown